### PR TITLE
Add Chinese language for this theme

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
can you help to review this?
@coliff @matcornic

File: exampleSite/config.toml [hunk 1]
```diff
@@ -73,3 +73,8 @@ weight = 20
 name = "<i class='fas fa-fw fa-bullhorn'></i> Crédits"
 url = "/credits"
 weight = 30
+
+[Languages.zh]
+title = "Hugo 主题的 Learn 文档"
```
Comment (matcornic) [inline exampleSite/config.toml]: Could you add shortcuts like for the other languages ?

Comment (LinuxSuRen) [reply exampleSite/config.toml]: Cool. I'll do it.

**Comment by LinuxSuRen** [comment]
Shortcuts were added. @matcornic

**Pull request merged**